src/features/movies/MoviesList/index.js

```diff
@@ -15,114 +15,64 @@ import {
-                        <ImageWrapper>
-                            <Image />
-                        </ImageWrapper>
-                        <ContentWrapper>
-                            <Title>Mulanhshsjsskskgfddddddddddd</Title>
-                            <StyledDate>
-                                2020
-                            </StyledDate>
-                            <TagsWrapper>
-                                <Tag>Action</Tag>
-                                <Tag>Thriller</Tag>
-                                <Tag>Action</Tag>
-                            </TagsWrapper>
-                            <RatingWrapper>
-                                <StarIcon />
-                                <Rate>7,8</Rate>
-                                <Votes>35 votes</Votes>
-                            </RatingWrapper>
-                        </ContentWrapper>
-                    </TileMovie>
-                    <TileMovie>
-                        <ImageWrapper>
-                            <Image />
-                        </ImageWrapper>
-                        <ContentWrapper>
-                            <Title>Mulan</Title>
-                            <StyledDate>
-                                2020
-                            </StyledDate>
-                            <TagsWrapper>
-                                <Tag>Action</Tag>
-                                <Tag>Thriller</Tag>
-                                <Tag>Action</Tag>
-                            </TagsWrapper>
-                            <RatingWrapper>
-                                <StarIcon />
-                                <Rate>7,8</Rate>
-                                <Votes>35 votes</Votes>
-                            </RatingWrapper>
-                        </ContentWrapper>
-                    </TileMovie>
-                </TilesContainer>
-            </MainWrapper>
+        <MainWrapper insideDetails={insideDetails}>
+            {!!loading ? (
+                <Loading />
+            ) : (
+                <>
+                    <Header>Popular movies</Header>
+                    <TilesContainer insideDetails={insideDetails}>
+                        {movies.map(movie =>
+                            <TileMovie
+                                key={movie.id}
+                            >
+                                <ImageWrapper>
+                                    <Image src={`https://image.tmdb.org/t/p/w400${movie.poster_path}`} alt="" />
+                                </ImageWrapper>
+                                <ContentWrapper>
+                                    <Title>{movie.original_title}</Title>
+                                    <StyledDate>
+                                        {movie.release_date.slice(0, 4)}
+                                    </StyledDate>
+                                    <TagsWrapper>
+                                        <Tag>Action</Tag>
```

Comment:
I suggest to also map the tags it got id and name so map by id & display name - movie.genres.id - something like that

Reply:
but we have created also fetchGenres etc.